model/riscv_types_zvk.sail

```diff
@@ -0,0 +1,81 @@
+/*=======================================================================================*/
```

Comment:
Since this file has mainly utility functions instead of types, we could call it `riscv_zvk_utils`?